controller.py

```diff
@@ -20,8 +20,9 @@ def logout(name: str) :
 
 def reserve_room(room_id: str):
   room_service = RoomService()
-  # room_service.reserve_room(room_id)
-  pass
+
+  # this doesn't stop users from reserving the room multiple times
+  rsvp_room = room_service.reserve_room(room_id)  
```

Comment:
Does this always return me a value? What if in the reservation the room was full, how do we handle that?

_Do we need to decouple our code a little bit now? (i.e. move our db logic away from our service?_